src/confd/yang/infix-services@2024-04-08.yang

```diff
@@ -0,0 +1,60 @@
+module infix-services {
+  yang-version 1.1;
+  namespace "urn:ietf:params:xml:ns:yang:infix-services";
+  prefix infix-svc;
+
+  organization "KernelKit";
+  contact      "kernelkit@googlegroups.com";
+  description  "Infix services, generic.";
+
+  revision 2024-04-08 {
+    description "Initial support for web services.";
+    reference "internal";
+  }
+  revision 2023-10-16 {
+    description "Drop SSDP support, mDNS-SD is now available in Windows 10.";
+    reference "internal";
+  }
+  revision 2023-08-22 {
+    description "Initial revision, add SSDP and mDNS-SD enable/disable only.";
+    reference "internal";
+  }
+
+  /*
+   * Data nodes
+   */
+
+  container mdns {
+    description "Advertise system and services over mDNS-SD, IPv4 and IPv6.";
+
+    leaf enabled {
+      description "Globally enable or disable mDNS/SD on all interfaces.";
+      type boolean;
+    }
+  }
+
+  container web {
```

Comment:
Should it be possible to disable `netbrowse`?

Reply:
I can add that too, but I wanted to keep down the scope a bit for the first drop.

Reply:
Added, I decided to call it just that, `netbrowse`.